tests/unit/config/test_configcommands.py

```diff
@@ -580,16 +583,16 @@ def test_with_sourcing(self, commands, config_stub, patch_editor):
 
     def test_config_py_with_sourcing(self, commands, config_stub, patch_editor, config_py_arg):
         assert config_stub.val.content.javascript.enabled
-        conf = 'c.content.javascript.enabled = False'
-        mock = patch_editor(conf)
+        conf = ['config.load_autoconfig(False)', 'c.content.javascript.enabled = False']
+        mock = patch_editor("\n".join(conf))
         commands.config_edit()
         mock.assert_called_once_with(unittest.mock.ANY)
         assert not config_stub.val.content.javascript.enabled
-        assert config_py_arg.read_text('utf-8').splitlines() == [conf]
+        assert config_py_arg.read_text('utf-8').splitlines() == conf
 
     def test_error(self, commands, config_stub, patch_editor, message_mock,
                    caplog):
-        patch_editor('c.foo = 42')
+        patch_editor('config.load_autoconfig(False)\nc.foo = 42')
```

Comment:
I'd prefer using `'\n'.join` here as well to make it a bit more readable (and consistent).